openwisp_notifications/types.py

```diff
@@ -0,0 +1,90 @@
+from django.core.exceptions import ImproperlyConfigured
+from openwisp_notifications.settings import ADDITIONAL_NOTIFICATION_TYPES
+
+DEFAULT_NOTIFICATION_TYPES = {
+    'default': {
+        'level': 'info',
+        'verb': 'default verb',
+        'name': 'Default Type',
+        'email_subject': '[{site}] Default Notification Subject',
+        'description': 'Default notification with {opts.verb} and {opts.level}',
+    },
+}
+
+
+def deep_merge_dicts(dict1, dict2):
+    result = dict1.copy()
+    for key, value in dict2.items():
+        if isinstance(value, dict):
+            node = result.get(key, {})
+            result[key] = deep_merge_dicts(node, value)
+        else:
+            result[key] = value
+    return result
```

Comment:
can you please open a PR for this in openwisp-utils? So we can pull this code from there. I will then change also openwisp-monitoring to use that one.